src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const weekDiscount = 50;
+  const threeDaysDiscount = 20;
+  const priceWithoutDiscout = days * dayCost;
+  let price = priceWithoutDiscout;
```

Comment:
Read the checklist. Name of variables should be Uppercase

Reply:
commit [3dd7caa](https://github.com/mate-academy/js_task-transportation-on-vacation/pull/3487/commits/3dd7caa8a62452623264a9a4b300e29df7f248c2)